docs/faq.md

```diff
@@ -37,6 +37,31 @@ $ xxd wallets/wallet.key
 
 Later this functionality will be included in `neofs-cli` directly.
 
+### How to create Neo wallet JSON file using a private key?
+
+You will need `neo-go` and `neofs-cli`.
+
+1. Get the WIF format of the private key
+
+```
+$ neofs-cli util keyer -key ./PATH/TO/YOUR/KEY | grep WIF | awk '{print $NF}' > temp_WIF
```

Comment:
Let's use key file from the repository as an example, e.g. service/http_gate/node.key

Reply:
added `./services/ir/01.key` as an existing example